pom.xml

```diff
@@ -1210,7 +1210,7 @@
 		<hibernateVersion>5.6.15.Final</hibernateVersion>
 		<hibernateSearchVersion>5.11.12.Final</hibernateSearchVersion>
 		<luceneVersion>5.5.5</luceneVersion>
-		<aspectjVersion>1.9.21.2</aspectjVersion>
```

Comment:
Is this change also related to the ticket you were working on?

Reply:
this one is related to the rebase process, to have the latest code from the repo. So. this changes come in

Reply:
If you do it properly, your pull request will not have this change because it is not related to the ticket.

Reply:
Did you just forget removing this change from the pull request?